### ISSUE TITLE
Poor image quality with last update

### ISSUE THREAD

**Issue by PTMaper** [mapillary/mapillary_issues#1948]
With last android app update pictures on mappilary look low res and blurry while looking ok on device, for Example https://www.mapillary.com/map/im/Gy7gzQB-pheEPPN_haCM3A/photo

**Comment by rsahlin** [comment]
Hi @PTMaper and thanks for reporting

Yes, this does not look ok - it could be the same artifacts as:
#1931 Which probably is due to automatic whitebalance/exposure


Best regards
/Richard

**Comment by PTMaper** [comment]
Yes this look similar. I have this problem with every Photo taken with my Gallaxy S7. It was OK bevor the last major update.

**Comment by rsahlin** [comment]
Hi

This could perhaps be related to missing JPEG image quality.
In the latest update, 2.22, I explicitly set the desired JPEG quality.
The update should be out through Google Play - could you please retest?

Best regards
/Richard

**Comment by PTMaper** [comment]
Still the same :( https://goo.gl/photos/3VZyo2LjqQpkFvLg9

**Comment by rsahlin** [comment]
Hi and thanks for checking so swiftly!

But that is a really good example photo of the artefact.
If I provide you with an APK link, will you be able to test if the artefact goes away when I change the auto settings?

/Richard

**Comment by PTMaper** [comment]
I think so. I will try.

**Comment by rsahlin** [comment]
Thanks - I will investigate and let you know as soon as I have something to test.
Hopefully within a day or so.
